proto/buf/alpha/registry/v1alpha1/studio_request.proto

```diff
@@ -0,0 +1,126 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.registry.v1alpha1;
+
+import "google/protobuf/timestamp.proto";
+
+// StudioRequestService exposes the functionality to manage favorited Requests
+// from Studio.
+service StudioRequestService {
+  // CreateStudioRequest registers a favorite Studio requests to the caller's
+  // BSR profile.
+  rpc CreateStudioRequest(CreateStudioRequestRequest) returns (CreateStudioRequestResponse) {}
+  // DeleteStudioRequest removes a favorite Studio request from the caller's BSR
+  // profile.
+  rpc DeleteStudioRequest(DeleteStudioRequestRequest) returns (DeleteStudioRequestResponse) {}
+  // ListStudioRequests shows the caller's favorited requests.
+  rpc ListStudioRequests(ListStudioRequestsRequest) returns (ListStudioRequestsResponse) {}
```

Comment:
based on your doc, did we want to also add `RenameStudioRequest`? (could also be post-mvp)

Reply:
Yeah! Almost forgot, thanks!